client/src/views/home/OrganizationListPage.vue

```diff
@@ -322,6 +447,14 @@ const filteredDevices = computed(() => {
       return 0;
     });
 });
+
+function toggleWebAccess(): void {
+  showWebAccess.value = !showWebAccess.value;
+}
+
+async function openMoreLink(): Promise<void> {
+  window.open(I18n.translate({ key: 'HomePage.webAccess.moreLink' }), '_blank');
```

Comment:
This is incorrect, you're forgetting the version.
Use https://github.com/Scille/parsec-cloud/blob/1bcdec7b545105509fd3294b09ff7791abd4ac81/client/src/services/environment.ts#L167, you can add a page.